### ISSUE TITLE
A11y high contrast/colorblind users

### ISSUE THREAD

**Issue by csutorasr** [box/box-ui-elements#2611]
**Environment:**

- Elements version:

**Desktop (please complete the following information):**

- OS: Windows 10
- Browser Chrome
- Version  91

**Steps to reproduce the problem:**

Enable high contrast mode. Try to differentiate the Browse your device button from the others visually.
The same goes for colorblind users.

**What is the expected behavior? (Screenshots can be helpful here)**

Underline or do some other modifications to make it distinguishable.

**What went wrong? (Screenshots, console logs, or HAR files can be helpful here)**

The design.

**Additional context**

The whole app should have better accessibility.
